### PR TITLE
Add GoldRush Guide

### PR DESCRIPTION
**Pull request by lukema95**
Related issue: #366 

## Summary by CodeRabbit

- **New Features**
	- Introduced the GoldRush API for accessing historical blockchain data.
	- Added comprehensive documentation for the GoldRush API, including setup instructions and usage examples for various programming languages.
- **Enhancements**
	- Expanded the existing services list to include GoldRush, providing users with more options for blockchain data management.

### CODE REVIEW

**Comment by lukema95** [comment]
@fadeev pls review. There is still a strange problem, that is, I can not see the `description` of the new document in the preview screen.

**Comment by fadeev** [comment]
> @fadeev pls review. There is still a strange problem, that is, I can not see the `description` of the new document in the preview card.

There is a typo in the filename`glodrush.mdx`, which doesn't match the key in `_meta.json`.

File: src/pages/developers/services/_meta.json [hunk 1]
```diff
@@ -22,5 +22,9 @@
   "subquery": {
     "title": "SubQuery",
     "description": "Open-source, flexible, multi-chain data indexer"
+  },
+  "goldrush": {
+    "title": "GoldRush API",
```
Comment (fadeev) [inline src/pages/developers/services/_meta.json:27]: Let's just keep the project name.

```suggestion
    "title": "GoldRush",
```

**Comment by fadeev** [comment]
Other than that, looks great!

**Comment by fadeev** [comment]
Please, merge `main` to fix build issues.

**Review by GMaiolo — APPROVED** [review]
utACK
LGTM

**Review by fadeev — APPROVED** [review]

**Pull request merged**